manifests/slave.pp

```diff
@@ -170,6 +176,16 @@
     }
   }
 
+  if $slave_args {
+    if is_array($slave_args) {
+      $_combined_slave_args = hiera_array('jenkins::slave::slave_args', $slave_args)
```

Comment:
I don't understand the need for a Hiera lookup though, you can already inject class parameters by defining:

```yaml
jenkins::slave::slave_args: "bippidyboppidy-boop"
```